src/components/archive.js

```diff
@@ -5,22 +5,43 @@ import Link from "./link";
 
 const environment = process.env.GATSBY_ENVIRONMENT_BRANCH;
 
-export default function Archive({ title, path = "", ...props }) {
+export default function Archive({
+  variant = "default",
+  title,
+  path = "",
+  ...props
+}) {
   const updatedPath = !path.startsWith("/") ? `/${path}` : path;
   const url = `https://github.com/EnterpriseDB/docs-archive/raw/${environment}${updatedPath}`;
 
+  return variant === "inverse" ? (
```

Comment:
Maybe instead of "inverse", "title-first"?

Reply:
I prefer to keep the `variant` prop as is (will update the variant name as suggested), I don't think this component needs to know if the primary link is a PDF or not.